client/src/pages/LogInForm.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+
+function LogInForm() {
+	const [emailInput, setEmailInput] = useState("");
+	const [passwordInput, setPasswordInput] = useState("");
+
+	// This variable handles shows success of the request to login with a boolean variable
+	const [backEndSuccess, setBackEndSuccess] = useState("");
+
+	const handleEmailChange = (e) => {
+		const email = e.target.value;
+		setEmailInput(email);
+	};
+
+	const handlePasswordChange = (e) => {
+		const password = e.target.value;
+		setPasswordInput(password);
+	};
+
+	async function postLogInDeatils(userData) {
+		try {
+			const response = await fetch("/mock-api", {
+				method: "POST",
+				headers: {
+					"Content-Type": "application/json",
+				},
+				body: JSON.stringify(userData),
+			});
+			const data = await response.json();
+			if (!response.ok) {
+				setBackEndSuccess(data.success);
```

Comment:
I meant the whole call to `setBackEndSuccess` is unnecessary

Reply:
How can we know what the error message is? For instance, if the registration fails, is it because of a wrong password or because the user has not been registered? Should I determine the reason based on the response status?

Reply:
This call to `setBackEndSuccess` is unnecessary because you immediately throw an error, which will trigger a different call to `setBackEndSuccess`.